design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,166 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#934](https://github.com/crossplaneio/crossplane/issues/934)
```

Comment:
This issue is closed as a dupe of https://github.com/crossplaneio/crossplane/issues/752, which is the tracking issue we are using.  Can you update this link to point to the open issue?

Reply:
Will do!